### PR TITLE
Link to service catalog documentation

### PR DESCRIPTION
**Pull request by zaro0508**
Provide a link to our public service catalog documentation from the
service catalog UI.

### CODE REVIEW

File: ec2/sc-ec2-linux-docker.yaml [hunk 1]
```diff
@@ -343,3 +343,6 @@ Outputs:
   EC2ConsoleURI:
     Description: 'Check your instance status with this link to the AWS Console'
     Value: !Sub "https://console.aws.amazon.com/ec2/v2/home?region=${AWS::Region}#Instances:search=${EC2Instance}"
+  Documentation:
+    Description: 'Service Catalog Documentation'
+    Value: "https://sagebionetworks.jira.com/wiki/spaces/SC/pages/938836322/Service+Catalog+Provisioning"
```
Comment (brucehoff) [inline ec2/sc-ec2-linux-docker.yaml:348]: The Confluence link is private, only accessible to Sage employees. Please use the Scroll Viewport URL which is public (and looks nicer):
https://help.sc.sageit.org/sc/Service-Catalog-Provisioning.938836322.html

File: ec2/sc-ec2-linux-jumpcloud-notebook.yaml [hunk 2]
```diff
@@ -541,3 +541,6 @@ Outputs:
       - ALBConnectionURI: !ImportValue
           'Fn::Sub': '${AWS::Region}-alb-notebook-access-ConnectionURI'
         Path: !Ref LinuxInstance
+  Documentation:
+    Description: 'Service Catalog Documentation'
+    Value: "https://sagebionetworks.jira.com/wiki/spaces/SC/pages/938836322/Service+Catalog+Provisioning"
```
Comment (brucehoff) [inline ec2/sc-ec2-linux-jumpcloud-notebook.yaml:546]: https://help.sc.sageit.org/sc/Service-Catalog-Provisioning.938836322.html

File: ec2/sc-ec2-windows-jumpcloud.yaml [hunk 3]
```diff
@@ -235,9 +235,9 @@ Outputs:
...
     Value: !Ref 'WindowsInstanceType'
-  ConnectionInstructions:
-    Description: 'Guidelines on connecting to instances'
-    Value: "https://sagebionetworks.jira.com/wiki/spaces/IT/pages/996376579/Connect+to+Provisioned+Instances"
   EC2ConsoleURI:
     Description: 'Check your instance status with this link to the AWS Console'
     Value: !Sub "https://console.aws.amazon.com/ec2/v2/home?region=${AWS::Region}#Instances:search=${WindowsInstance}"
+  Documentation:
+    Description: 'Service Catalog Documentation'
+    Value: "https://sagebionetworks.jira.com/wiki/spaces/SC/pages/938836322/Service+Catalog+Provisioning"
```
Comment (brucehoff) [inline ec2/sc-ec2-windows-jumpcloud.yaml:243]: https://help.sc.sageit.org/sc/Service-Catalog-Provisioning.938836322.html

File: s3/sc-s3-encrypted-ra.yaml [hunk 4]
```diff
@@ -84,3 +84,6 @@ Outputs:
   BucketUrl:
     Description: 'View the S3 Bucket in the AWS Console'
     Value: !Sub 'https://console.aws.amazon.com/s3/home?region=${AWS::Region}&bucket=${S3Bucket}'
+  Documentation:
+    Description: 'Service Catalog Documentation'
+    Value: "https://sagebionetworks.jira.com/wiki/spaces/SC/pages/938836322/Service+Catalog+Provisioning"
```
Comment (brucehoff) [inline s3/sc-s3-encrypted-ra.yaml:89]: https://help.sc.sageit.org/sc/Service-Catalog-Provisioning.938836322.html

File: s3/sc-s3-synapse-ra.yaml [hunk 5]
```diff
@@ -124,3 +124,6 @@ Outputs:
   BucketUrl:
     Description: 'View the S3 Bucket in the AWS Console'
     Value: !Sub 'https://console.aws.amazon.com/s3/home?region=${AWS::Region}&bucket=${S3Bucket}'
+  Documentation:
+    Description: 'Service Catalog Documentation'
+    Value: "https://sagebionetworks.jira.com/wiki/spaces/SC/pages/938836322/Service+Catalog+Provisioning"
```
Comment (brucehoff) [inline s3/sc-s3-synapse-ra.yaml:129]: https://help.sc.sageit.org/sc/Service-Catalog-Provisioning.938836322.html

**Review by brucehoff — CHANGES_REQUESTED** [review]

File: ec2/sc-ec2-linux-docker.yaml (hunk 1 above)
Comment (zaro0508) [reply ec2/sc-ec2-linux-docker.yaml]: thanks, atlassian is so confusing. anyways, it's fixed now.

**Review by brucehoff — APPROVED** [review]

**Pull request merged**